client/my-sites/site-settings/navigation.jsx

```diff
@@ -56,6 +57,14 @@ export class SiteSettingsNavigation extends Component {
 						{ strings.general }
 					</NavItem>
 
+					<NavItem
```

Comment:
I also feel it's a little weird to have it before Writing _but_:

- It might be because I'm already used to it.
- Performance is arguably more important than the writing set up.

What do you think about this point @joanrho @jeffgolenski ?

Reply:
Given customer feedback about how important site performance is to them (specifically with Jetpack), I'd think Performance should be the first tab immediately after General.